src/js/samples/components/Item/EndIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "styled-components";
+import { Icon, Loader } from "../../../base";
+
+const SampleIconContainer = styled.div`
```

Comment:
There is a convention for naming in Styled Components for this specific situation.

1. The exported component (eg. `SampleItemEndIcon`) is styled by using a Styled Component (eg. StyledSampleItemEndIcon`) as the top-level component in its return block.
2. The Styled Component is used **only** to style the exported component. It is not reused in other components.

This situation is super common; hence the convention.

```suggestion
const StyledSampleItemEndIcon = styled.div`
```